draft-ietf-tls-tls13.md

```diff
@@ -4302,7 +4305,7 @@ applicable features:
 
   * "supported_versions" is REQUIRED for all ClientHello messages.
   * "signature_algorithms" is REQUIRED for certificate authentication.
-  * "supported_groups" and "key_share" are REQUIRED for DHE or ECDHE key exchange.
```

Comment:
This is correct as-is.
A && B are required for D || C
if D || C, must A && B
NOT just if D && C, must A && B